packages/arb-token-bridge-ui/src/components/TokenModal/TokenImportModal.tsx

```diff
@@ -129,72 +112,81 @@ export function TokenImportModal({
     }
   }, [status])
 
+  const getL1TokenData = useCallback(
+    async (eitherL1OrL2Address: string) => {
```

Comment:
personal stylistic preference is to do a ` if(!bridge)return` upfront and avoid the `?`s

Reply:
Definitely, I dislike optional chaining myself. Missed it here.